doc/user/content/ingest-data/appendix/postgres-debezium.md

```diff
@@ -4,13 +4,22 @@ description: "How to propagate Change Data Capture (CDC) data from a PostgreSQL
 aliases:
   - /connect-sources/cdc-postgres-kafka-debezium/
   - /ingest-data/cdc-postgres-kafka-debezium/
+  - /ingest-data/postgres/debezium/
+
 menu:
   main:
-    parent: "postgresql"
-    name: "Using Kafka and Debezium"
+    parent: "ingest-data-appendix"
+    name: "PosgreSQL: Using Kafka and Debezium"
     identifier: "pg-dbz"
 ---
 
+{{< tip >}}
+- As an alternative to using Kafka and Debezium, Materialize also provides
+  native support for PostgreSQL.
```

Comment:
Once we sort out our docs frontmatter so that there is a landing page under Ingest data/PostgreSQL/, we can update this to link there.